### DISCUSSION TITLE
Private repo to a team in an organization

### DISCUSSION THREAD

**Discussion by wkjung** [community/community#24007] [New to GitHub]
<p>Hi.<br>
I would like to make a repo only visible to a team in an organization, but still leaving the base permission of the organization to be “read”.<br>
Is that possible? From reading the manuals, it seems it is mandatory to set the base permission for the organization to “None” for this situation.<br>
Is there any workaround or am I missing something?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/wkjung">@wkjung</a> :wave:</p>
<p>Welcome to the GitHub Community!</p>
<p>This particular functionality is not present in our product. That level of specificity would require a new feature.</p>
<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>
